crates/iota-sdk-ffi/src/crypto/simple.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_crypto::{Signer, Verifier};
+use iota_types::SignatureScheme;
+
+use crate::{
+    error::Result,
+    types::{crypto::multisig::MultisigMemberPublicKey, signature::SimpleSignature},
+};
+
+#[derive(derive_more::From, uniffi::Object)]
+pub struct SimpleVerifier(iota_crypto::simple::SimpleVerifier);
+
+#[uniffi::export]
+impl SimpleVerifier {
+    #[uniffi::constructor]
+    pub fn new() -> Self {
+        Self(iota_crypto::simple::SimpleVerifier)
+    }
+
+    pub fn verify(&self, message: &[u8], signature: &SimpleSignature) -> Result<()> {
+        Ok(self.0.verify(message, &signature.0)?)
+    }
+}
+
+#[derive(derive_more::From, uniffi::Object)]
+pub struct SimpleKeypair(iota_crypto::simple::SimpleKeypair);
+
+#[uniffi::export]
+impl SimpleKeypair {
```

Comment:
Do we not want constructors for the variants?

Reply:
Which variants would those be?